pkg/kube/kube_test.go

```diff
@@ -178,4 +178,41 @@ var _ = Describe("GetApplication", func() {
 		Expect(err).ShouldNot(HaveOccurred())
 		Expect(out.Spec.Path).To(Equal("some/path"))
 	})
+
+	It("get all applications", func() {
+
+		appsList := &wego.ApplicationList{Items: []wego.Application{
+			{
+				Spec: wego.ApplicationSpec{
+					Path: "some/path0",
+					URL:  "example.com/some-org/some-repo0",
+				},
+			},
+			{
+				Spec: wego.ApplicationSpec{
+					Path: "some/path1",
+					URL:  "example.com/some-org/some-repo1",
+				},
+			},
+		}}
+
+		res, err := json.Marshal(appsList)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		runner.RunStub = func(cmd string, args ...string) ([]byte, error) {
+			return res, nil
+		}
+
+		apps, err := kubeClient.GetApplications("wego-system")
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(2).To(Equal(len(*apps)))
+
+		for i, a := range appsList.Items {
+			apps := *apps
```

Comment:
Another nit, but this seems like an unnecessarily obscure usage. I feel like we should avoid shadowing a variable with a different type

Reply:
it didn't seem right to me either 🙂. I forgot to look up for a different approach. This way allowed me to fix an issue with the compiler when I tried to use `*apps[i].name`. I would deeply appreciate a suggestion for the right way. For this approach in particular I guess I also missed to have this assignation outside of the `for` 🤔

Reply:
found it 🙂

Reply:
:+1: